manifests/params.pp

```diff
@@ -31,6 +31,12 @@
     $service_enable = true
   }
 
+  if ($::operatingsystem == 'Ubuntu' and $::operatingsystemrelease in ['15.04','15.10','16.04','16.10']) or ($::operatingsystem == 'Debian' and $::operatingsystemrelease == '8') {
```

Comment:
This is not a core fact, right? So we would need a dependency for puppet-systemd... I'm not so sure if this is a good decision? As of Ubuntu 16.04 with facter 1.7.5 this fact is not present.